specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1368,6 +1368,379 @@
+            },
+            "description": "The parameters to provide for the current SQL container."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The SQL container create operation was successful",
+            "schema": {
+              "$ref": "#/definitions/SqlContainer"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/apis/sql/databases/{databaseRid}/containers/{containerRid}": {
+      "get": {
+        "operationId": "DatabaseAccounts_GetSqlContainer",
+        "x-ms-examples": {
+          "CosmosDBSqlContainerGet": { "$ref": "./examples/CosmosDBSqlContainerGet.json" }
+        },
+        "description": "Gets the SQL container under an existing Azure Cosmos DB database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/containerRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The SQL container property was retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/SqlContainer"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "DatabaseAccounts_UpdateSqlContainer",
+        "x-ms-examples": {
+          "CosmosDBSqlContainerReplace": { "$ref": "./examples/CosmosDBSqlContainerReplace.json" }
+        },
+        "description": "Updates an existing Azure Cosmos DB SQL container",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/containerRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "updateSqlContainerParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SqlContainerCreateUpdateParameters"
+            },
+            "description": "The parameters to provide for the current SQL container."
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
is this not async?

Reply:
This is async operation, but I found existing example of database account is:
          "200": {
            "description": "The database account create or update operation will complete asynchronously.",
            "schema": {
              "$ref": "#/definitions/DatabaseAccount"
            }
          }
So what should be the correct response?

Reply:
An async operation should have `"x-ms-long-running-operation": true` and a 202 response (in addtion to 200 - for noop- and default error). If this is a syncronous operation in the service, it is fine as is.